web_src/js/components/DiffFileTree.vue

```diff
@@ -94,6 +94,7 @@ export default {
       // Merge folders with just a folder as children in order to
       // reduce the depth of our tree.
       mergeChildIfOnlyOneDir(result);
+      this.isIncomplete = pageData.diffFileInfo.isIncomplete;
```

Comment:
Do not change value in a `computed` function. It breaks the reactive system

Reply:
Is it a use case for a store?

Reply:
Yup, https://vuejs.org/guide/essentials/computed.html#best-practices

Reply:
done in [35422ca](https://github.com/go-gitea/gitea/pull/24983/commits/35422cab9693ff79bf8e2a1bd72f1e4029cb117b)